### PR TITLE
Empty Authority field implies local UUri

### PR DESCRIPTION
**Pull request by AnotherDaniel**
UUris with empty Authority are considered to be local - this was missing in the is_remote_uuri check.

### CODE REVIEW

File: src/uri.rs [hunk 1]
```diff
@@ -406,7 +407,8 @@ impl UUri {
     /// assert!(authority_a.is_remote_authority(&authority_b));
     /// ````
     pub fn is_remote_authority(&self, other_uri: &UUri) -> bool {
-        self.authority_name != WILDCARD_AUTHORITY
+        !self.authority_name.is_empty()
```
Comment (sophokles73) [inline src/uri.rs:410]: a unit test would be nice, also covering the other cases ...

Comment (AnotherDaniel) [reply src/uri.rs:418]: Done

File: src/uri.rs [hunk 2]
```diff
@@ -404,9 +405,18 @@ impl UUri {
     /// let authority_a = UUri::from_str("up://Authority.A/100A/1/0").unwrap();
     /// let authority_b = UUri::from_str("up://Authority.B/200B/2/20").unwrap();
     /// assert!(authority_a.is_remote_authority(&authority_b));
+    ///
+    /// let authority_local = UUri::from_str("up:///100A/1/0").unwrap();
+    /// assert!(!authority_local.is_remote_authority(&authority_a));
+    ///
+    /// let authority_wildcard = UUri::from_str("up://*/100A/1/0").unwrap();
+    /// assert!(!authority_local.is_remote_authority(&authority_a));
```
Comment (sophokles73) [inline src/uri.rs:413]: this seems to be a duplicate of 410

Comment (AnotherDaniel) [reply src/uri.rs]: Thx for spotting this - copy&pasty...

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**